web-services-sample-app/GetWeatherVC.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ViewController.swift
+//  web-services-sample-app
+//
+//  Created by webonise on 29/09/16.
+//  Copyright © 2016 webonise. All rights reserved.
+//
+
+import UIKit
+
+class GetWeatherVC: UIViewController {
+    
+    @IBOutlet var textFieldEnterCityName: UITextField!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+    }
+
+    /** This function checks for validations whether the user has added only whitespaces in the text field
+     * \returns Nothing
+     */
+    func isValid(isValidString: NSString) -> Bool {
```

Comment:
1) Provide proper naming for functions, 'isValid' does not suggest much about what the method does.Same goes for the parameter name 'isValidString'
2) In the documentation use this format:
- paramter parameterName: description
- returns: Description
3) The method returns Bool as a return value.The Documentation reads returns Nothing.Update this